packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -164,6 +164,18 @@
 							</div>
 						</div>
 					</section>
+
+					<section class="form-section">
+						<template v-if="interventionPolicy">
```

Comment:
would it still look fine if you put the v-if in the section?

Reply:
yes!